sdks/go/test/run_validatesrunner_tests.sh

```diff
@@ -430,8 +424,7 @@ echo ">>> RUNNING $RUNNER integration tests with pipeline options: $ARGS"
 cd ../..
 
 if [[ "$RUNNER" == "dataflow" ]]; then
-  # Delete the container locally and remotely
-  docker rmi $CONTAINER:$TAG || echo "Failed to remove container"
+  # Delete the container remotely
```

Comment:
we could still try delete locally as best effort, if that would help non-arm path.

Reply:
If there is no local container found, keep this line will cause the task to fail with the message saying "Failed to remove container". Is there a way to try to clean the container locally, but if not found, we only echo the message but no error thrown?

Reply:
that is what should be happening there is an OR (||), so the command shouldn't fail, only error message is shown. We could adjust the error message text though.

Reply:
I see! Just changed the error message.